### PR TITLE
Fix license spelling

### PR DESCRIPTION
**Pull request by bgerrity**
Fixed the inconsistent 🪪  spelling in the repo.

### CODE REVIEW

File: cmd/tools/gendynamicconfig/main.go [hunk 1]
```diff
@@ -329,7 +329,7 @@ func readLicenseFile(path string) string {
 }
 
 func main() {
-	licenseFlag := flag.String("licence_file", "../../LICENSE", "path to license to copy into header")
+	licenseFlag := flag.String("License_file", "../../LICENSE", "path to license to copy into header")
```
Comment (dnr) [inline cmd/tools/gendynamicconfig/main.go:332]: I think these shouldn't be capitalized?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,134 +1,163 @@
...
 
 **Note:** All contributors also need to fill out the [Temporal Contributor License Agreement](docs/development/temporal-cla.md) before we can merge in any of your changes.
 
 ## Prerequisites
 
-### Build prerequisites 
-* [Go Lang](https://golang.org/) (minimum version required is 1.19):
+### Build prerequisites
+
+- [Go Lang](https://golang.org/) (minimum version required is 1.19):
```
Comment (alexshtin) [inline CONTRIBUTING.md:11]: We are on 1.22 now, but I think it would be better to remove minimum version at all and replace link with "https://go.dev".

**Review by alexshtin — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (alexshtin) [reply CONTRIBUTING.md:11]: I will handle this in separate PR.

File: cmd/tools/copyright/licensegen.go [hunk 3]
```diff
@@ -59,7 +59,7 @@ var (
 	// directories to be excluded
 	dirBlocklist = []string{".gen/", ".git/", ".vscode/", ".idea/"}
 	// default perms for the newly created files
-	defaultFilePerms = os.FileMode(0644)
+	defaultFilePerms = os.FileMode(0o644)
```
Comment (alexshtin) [inline cmd/tools/copyright/licensegen.go:62]: :open_mouth:

Comment (bgerrity) [reply cmd/tools/copyright/licensegen.go:62]: Ride or die opinionated linters.

**Pull request merged**